Sources/TripKit/model/API/RoutingAPIModel.swift

```diff
@@ -35,6 +35,7 @@ extension TKAPI {
   public struct Trip: Codable, Hashable {
     @ISO8601OrSecondsSince1970 public var depart: Date
     @ISO8601OrSecondsSince1970 public var arrive: Date
+    public let hideExactTimes: Bool?
```

Comment:
Could just be a `Bool` and we default it to `false`. That gives us just two states to deal with:

```suggestion
    @DefaultFalse public var hideExactTimes: Bool
```